teuthology/task/ansible.py

```diff
@@ -60,6 +60,8 @@ class Ansible(Task):
                     not, we generate a temporary file to use.
         tags:       A string including any (comma-separated) tags to be passed
                     directly to ansible-playbook.
+        vars:       A dict of vars to be passed to ansible-playbook via the
+                    --extra-vars flag
```

Comment:
Might be worth mentioning the possible values for a var are a yaml string, list or hash.